Dockerfile-ubuntu2004-dpdk2011

```diff
@@ -10,17 +11,18 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 RUN apt-get update && apt-get -y install \
   apt-utils \
   build-essential \
+  cmake \
   curl \
   git \
-  tar \
-  wget \
-  xz-utils \
-  libpcap-dev \
   libnuma-dev \
-  cmake \
+  libpcap-dev \
   meson \
-  linux-headers-$(uname -r)
+  tar \
+  wget \
+  xz-utils
 
-RUN wget http://fast.dpdk.org/rel/dpdk-20.11.3.tar.gz
-RUN tar xvzf dpdk-20.11.3.tar.gz
-RUN mv dpdk-stable-20.11.3 dpdk && cd dpdk && meson build && cd build && ninja && ninja install && ldconfig
+RUN wget http://fast.dpdk.org/rel/dpdk-${DPDK_VERSION}.tar.gz \
+  && tar xvzf dpdk-${DPDK_VERSION}.tar.gz \
+  && mv dpdk-stable-${DPDK_VERSION} dpdk && cd dpdk \
+  && meson build && cd build && ninja && ninja install && ldconfig \
+  && cd ../.. && rm -rf dpdk*
```

Comment:
I'm not sure we want to remove the `dpdk` directory, otherwise this check will fail in PcapPlusPlus configuration:
https://github.com/seladb/PcapPlusPlus/blob/5b5ef5a2764ac13a55e48f57e49a29bcaf53b880/configure-linux.sh#L167

Reply:
@seladb ninja install will install the libs in /usr/local/lib. So DPDK_HOME dir should be set to /usr/local/. 

If you want we can explicitly install the libs in a folder like /opt/dpdk-20.11 for example. So DPDK_HOME should be set to this value. 

With CMake if the libs are installed in /usr or /usr/local CMake will find the libs automatically without setting it.